ber.go

```diff
@@ -5,10 +5,14 @@ import (
 	"errors"
 	"fmt"
 	"io"
+	"math"
 	"os"
 	"reflect"
 )
 
+// MaxPacketLengthBytes specifies the maximum allowed packet size when calling ReadPacket or DecodePacket.
+var MaxPacketLengthBytes int = math.MaxInt32
```

Comment:
make this type int64 for future compatibility

Reply:
@liggitt I'll try, not sure how this will work when `GOARCH=386`, may need to convert it.

Somewhat related, in `.travis.yml` you can test other OS's/architectures. I think this might be useful now since we're using `int64` and a builtin. Travis' `env` builds a matrix with all the Go versions, do you have an idea which `GOOS` / `GOARCH` combinations you'd like to target and which Go versions to keep?

```
env:
  - GOOS=linux GOARCH=amd64
  - GOOS=windows GOARCH=amd64
  - GOOS=linux GOARCH=386
  - GOOS=windows GOARCH=386
```

Reply:
We've already had a bug in the past with 386 arch; so I think adding that to the test matrix would be a win.

It's appealing to me to keep testing as far back as we do (1.2).

Reply:
@johnweldon Oops, missed this comment before pushing `.travis.yml`. Let's see how this one goes (I know it'll build Windows binaries but not sure it can run tests) then I'll add back the ones I dropped.